electroncash/qrreaders/zxing.py

```diff
@@ -0,0 +1,74 @@
+# Electron Cash - lightweight Bitcoin client
+# Copyright (C) 2023 Axel Gembe <derago@gmail.com>
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation files
+# (the "Software"), to deal in the Software without restriction,
+# including without limitation the rights to use, copy, modify, merge,
+# publish, distribute, sublicense, and/or sell copies of the Software,
+# and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import ctypes
+from typing import List
+
+import zxingcpp
```

Comment:
As a courtesy for users running from dev source.. might be helpful to give them this message. What do you think? I know when I first ran this I had to like scour the requirements-binaries.txt to get the version we need and so perhaps just having it printed in the output may be helpful. What do you think?

```suggestion
try:
    import zxingcpp
except ModuleNotFoundError:
    import sys
    print("\n\nzxing-cpp module missing. Please try installing it with: python3 -m pip install 'zxing-cpp>=2.2.0'"
          " --user\n\n", file=sys.stderr)
    raise
```

Reply:
Added that as well as handling the `ModuleNotFoundError` for detecting which QR readers are available.